src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -638,6 +638,7 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
   SendOnAppRegisteredNotificationToHMI(
       *(application.get()), resumption, need_restore_vr);
 
+  GetPolicyHandler().AddApplication(application->policy_app_id());
```

Comment:
@dev-gh please consifer that I checked ATF script from #1137 and it is passed. 
OnAppRegistered and  OnStatusUpdate hanle thatir positions.